lib/arel/visitors/to_sql.rb

```diff
@@ -131,6 +131,7 @@ def visit_Arel_Nodes_InsertStatement o, collector
         end
 
         if o.values
+          collector << " VALUES"
```

Comment:
Thanks @kamipo. I reveted and added a regression test. 5db56a513286814991c27000af2c0243cc19d1e2